index.html

```diff
@@ -65,20 +65,20 @@
     </linkedin>
     </social_icons>
 
-    <headline>
-        <primary_text>Hey There, I'm Alex.</primary_text>
-        <supporting_text>I'm a Software Developer.</supporting_text>
-        <description>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</description>
+    <headline id="headline">
+        <primary_text id="primary_text">Hey There, I'm Alex.</primary_text>
+        <supporting_text id="supporting_text">I'm a Software Developer.</supporting_text>
+        <description id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</description>
     </headline>
 
-    <illustration_header>
+    <illustration_header id="illustration_header">
         
-        <illustration>
+        <illustration id="illustration">
             <img src="./assets/Illustration-header.svg" alt=""> </img>
         </illustration>
```

Comment:
- [ ] Kindly make sure that section has the proper background image (decorations are always background images, not `img` tags).If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.